loaders/android/AndroidManifest.xml.in

```diff
@@ -22,6 +22,6 @@
     @ANDROID_XML_SERVICES@
   </application>
 @ANDROID_XML_PERMISSIONS@
-<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:required="true" />
```

Comment:
android:required exists in [uses-feature](https://docs.huihoo.com/android/4.2///guide/topics/manifest/uses-feature-element.html) but not [uses-permission](https://developer.android.com/guide/topics/manifest/uses-permission-element)?